tests/conftest.py

```diff
@@ -26,6 +27,116 @@ def database(request, pymongo_client):
     return database
 
 
+def batch(
+    batch_id = "201860",
+    sequencing_date = "2022-03-10",
+    stdev_13= 0.0095,
+    stdev_18 = 0.0035,
+    stdev_21 = 0.0085,
+):
+    """A batch mock"""
+
+    return {
+        "_id": batch_id,
+        "SequencingDate": sequencing_date,
+        "Median_13": 0.994623404680424,
+        "Median_18": 1.01950547134618,
+        "Median_21": 1.00710552415657,
+        "Median_X": 0.8408228472301729,
+        "Median_Y": 0.00011224982090899999,
+        "Stdev_13": stdev_13,
+        "Stdev_18": stdev_18,
+        "Stdev_21": stdev_21,
+        "Stdev_X": 0.029800076293786,
+        "Stdev_Y": 0.0000653186791196846,
+    }
+
+
+def sample(
```

Comment:
This fixture is not used in the tests of this PR. Do you need it for later?

If so, since perhaps create a list with chromosome values [1,..22] and instantiate the variables using this list? Otherwise it's quite repetitive. And do you really care about what type of values these variables have in the test?